src/test/java/com/amazon/sqs/javamessaging/MessageListenerConcurrentOperationTest.java

```diff
@@ -292,7 +288,7 @@ public void verify() {
 
     public void testConcurrentExecution(final MessageListener msgListener, final ConcurrentOperation operation) throws JMSException, InterruptedException {
 
-        long start = System.currentTimeMillis();
+        //long start = System.currentTimeMillis();
```

Comment:
Is this intentional?

Reply:
Yes, the `start` never used in this test. I can remove this comment as it is already pretty straightforward.